run_pylint.sh

```diff
@@ -22,11 +22,24 @@ then
     fi
 fi
 
-TMPFILENAME="/tmp/run_pylint_$RANDOM"
+# Tests need to include autotest modules; make sure we can access them.
+if [ -z "$AUTOTEST_PATH" ]
+then
+    python -c 'import autotest'
+    if [ "$?" -ne "0" ]
+    then
+        echo "ERROR: autotest module won't load or " \
+             "AUTOTEST_PATH env. var. is not set"
+        exit 1
+    fi
+fi
+
+TMPFILENAME="$(mktemp --suffix=run_pylint)"
```

Comment:
(Very minor): how about `mktemp --tmpdir $(basename $0).XXXXXXX` ? That (1) removes hardcoded run_pylint, (2) puts the script name at the beginning instead of at the end so if there are /tmp droppings -- which there shouldn't be, thanks to `cleanup`, but Murphy's Law -- they are easier to identify. Or if not, how about prepending a dot so the suffix doesn't run-on with the XXXXX?